com.unity.multiplayer.mlapi/Runtime/Core/NetworkManager.cs

```diff
@@ -1550,6 +1550,11 @@ private void SyncTime()
 
         internal void HandleApproval(ulong ownerClientId, bool createPlayerObject, uint? playerPrefabHash, bool approved, Vector3? position, Quaternion? rotation)
         {
+            if (playerPrefabHash == null && createPlayerObject)
+            {
+                playerPrefabHash = NetworkConfig.PlayerPrefab.GetComponent<NetworkObject>().GlobalObjectIdHash;
+            }
```

Comment:
should we move this inside `if (createPlayerObject) { ... }` block below to simply logic here?

I'm imagining something like this:
```
if (createPlayerObject)
{
    if (playerPrefabHash == null)
    {
        playerPrefabHash = NetworkConfig.PlayerPrefab.GetComponent<...
    }

    var networkObject = SpawnManager.CreateLocalNetworkObject(...
```

Reply:
Completely reverted back to original code.